com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -582,7 +583,8 @@ private void validateBuildResourceMapping() throws CompileExceptionError {
             List<IResource> outputs = t.getOutputs();
             for (IResource output : outputs) {
                 String outStr = output.toString();
-                if (build_map.containsKey(outStr)) {
+                boolean isGenerated = outStr.contains("_generated_");
+                if (build_map.containsKey(outStr) && !isGenerated) {
```

Comment:
This is a check to make sure tasks don't have the same output file (where it's not wanted)